packages/ruby/spec/readme/har/response_serializer_spec.rb

```diff
@@ -105,6 +105,63 @@
       expect(json.dig(:content, :size)).to eq 0
       expect(json.dig(:content, :mimeType)).to eq ""
     end
+
+    it "handles multiple JSON mime types" do
+      json_mime_types = [
+        "application/json",
+        "application/x-json",
+        "text/json",
+        "text/x-json",
+        "+json"
+      ]
+      request = build_request
+      body = {result: "value"}.to_json
+
+      json_mime_types.each do |mime_type|
+        response = build_response(
+          status: 200,
+          content_type: mime_type,
+          body: body
+        )
+        serializer = Readme::Har::ResponseSerializer.new(
+          request,
+          response,
+          Filter.for
+        )
+        json = serializer.as_json
+
+        expect(json[:content][:text]).to eq body
+        expect(json[:content][:mimeType]).to eq mime_type
+      end
+    end
+
+    it "handles a text/plain response without a Content-Length header" do
+      request = build_request
+      response = HttpResponse.from_parts(
+        200,
+        {"Content-Type" => "text/plain"},
+        {key: "value"}.to_json
+      )
+
+      serializer = Readme::Har::ResponseSerializer.new(request, response, Filter.for)
+      json = serializer.as_json
+
+      expect(json).to match_json_schema("response")
+    end
+
+    it "handles an application/json response without a Content-Length header" do
+      request = build_request
+      response = HttpResponse.from_parts(
+        200,
+        {"Content-Type" => "application/json"},
+        {key: "value"}.to_json
+      )
+
+      serializer = Readme::Har::ResponseSerializer.new(request, response, Filter.for)
+      json = serializer.as_json
+
+      expect(json).to match_json_schema("response")
+    end
```

Comment:
I think these cases don't really fit here? The serializer doesn't know or care about the `Content-Type` header. All it knows is that a collaborator responds to `content_length`. That might already be exercised by the other tests?

Reply:
Yup, good call. This is covered by the unit tests for the response object